src/pevm.rs

```diff
@@ -42,8 +44,11 @@ pub enum PevmError {
     UnreachableError,
 }
 
-/// Execution result of PEVM.
-pub type PevmResult = Result<Vec<ResultAndState>, PevmError>;
+/// Execution result of a transaction
+pub type PevmExecutionResult = (Receipt, EvmState);
```

Comment:
Looking back, `PevmTxExecutionResult` is a better name. Let's also turn this into a `struct`.

Reply:
fixed